src/squeue.rs

```diff
@@ -178,55 +175,15 @@ impl SubmissionQueue {
     pub fn is_full(&self) -> bool {
         self.len() == self.capacity()
     }
-
-    /// Take a snapshot of the submission queue.
-    #[inline]
-    pub fn available(&mut self) -> AvailableQueue<'_> {
-        unsafe {
-            AvailableQueue {
-                head: (*self.head).load(atomic::Ordering::Acquire),
-                tail: unsync_load(self.tail),
-                ring_mask: self.ring_mask.read(),
-                ring_entries: self.ring_entries.read(),
-                queue: self,
-            }
-        }
-    }
 }
 
-impl AvailableQueue<'_> {
```

Comment:
I still want to keep `AvailableQueue`, which has better performance in theory and is not complicated.